cli/man/TEMPLATE.1.md.example

```diff
@@ -0,0 +1,69 @@
+% COMMAND(1) Cargill, Incorporated | Splinter Commands
+
+<!--
+This file should be named the full command (with dashes between subcommands)
+with an extension of `1.md`. For example `splinter-cert-generate.1.md` is the
+markdown file that will generated into the man page for
```

Comment:
Please change to "that generates the man page for..."

Also (nitpick): Markdown is written with a capital M.

Reply:
updated